ERCS/erc-7558.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 7558
+title: Minimal Lockable Range NFTs
+description: A minimal interface for soulbinding multiple ERC-721 NFTs
+author: Hans Bergren (@0xCLARITY)
+discussions-to: https://ethereum-magicians.org/t/erc-7558-minimal-lockable-range-nfts/16668
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-15
+requires: 165, 721
```

Comment:
```suggestion
requires: 165, 721, 5192
```

EIP-5192 is definitely required reading if the interface here extends `IERC5192`.